### PR TITLE
Fix #21 added bivio_brew_install; fixed bivio_pyenv_install for Darwin

### LINKED ISSUE #21: pyenv on macos needs brew

### PR DESCRIPTION
**Pull request by robnagler**

### CODE REVIEW

**Comment by robnagler** [comment]
hold off reviewing, darwin still not quite fixed.

**Comment by robnagler** [comment]
@e-carlin when you get a chance, please review.

File: bashrc.d/zz-20-dev.sh [hunk 1]
```diff
@@ -85,6 +85,36 @@ if [[ -d $PYENV_ROOT/bin ]]; then
...
+xcode-select --install
+' 1>&2
+        return 1
+    fi
+    local d="$HOME/brew"
+    if ! mkdir "$d"; then
+        echo "$d already exists; homebrew is already installed" 1>&2
+        return 1
+    fi
+    curl -L -s -S https://github.com/Homebrew/brew/tarball/master | tar xz --strip 1 -C "$d"
```
Comment (e-carlin) [inline bashrc.d/zz-20-dev.sh:110]: Why not use the install mechanism (and dir) that brew has listed on their site? https://brew.sh/

**Review by e-carlin — APPROVED** [review]
Looks fine. One minor comment. I have brew installed already and I don't know how to uninstall it so I didn't actually run the code.

File: bashrc.d/zz-20-dev.sh (hunk 1 above)
Comment (robnagler) [reply bashrc.d/zz-20-dev.sh:110]: brew.sh requests root access, but that won't workin a container or unattended.

Comment (e-carlin) [reply bashrc.d/zz-20-dev.sh:110]: Ah, thanks.

**Pull request merged**